src/main/java/org/jenkinsci/plugins/ghprb/Ghprb.java

```diff
@@ -406,6 +407,10 @@ private static String createCredentials(String serverAPIUrl, StandardCredentials
         }
         
         specifications.add(new SchemeSpecification(serverUri.getScheme()));
+        String path = serverUri.getPath();
+        if (StringUtils.isEmpty(path)) {
+            path = "/";
+        }
         specifications.add(new PathSpecification(serverUri.getPath(), null, false));
```

Comment:
this didn't actually fix the issue - it's not using the new path you created.

I have a few other suggestions. I'll send a PR shortly.